packages/wallet-core/src/swaps/thorchain/ThorchainSwapProvider.ts

```diff
@@ -188,8 +188,10 @@ class ThorchainSwapProvider extends SwapProvider {
 
   async _getTransaction(hash: string): Promise<ThorchainTransactionResponse | null> {
     try {
-      return await this.thorchainAPIErrorParser.wrapAsync(() => this._httpClient.nodeGet(`/thorchain/tx/${hash}`), {
-        txHash: hash,
+      const hasOx = hash.charAt(0) + hash.charAt(1) === '0x'
+      const transactionHash = hasOx ? hash.slice(2) : hash
+      return await this.thorchainAPIErrorParser.wrapAsync(() => this._httpClient.nodeGet(`/thorchain/tx/${transactionHash}`), {
+        txHash: transactionHash,
```

Comment:
@mpwanyi256 Is there a reason you're taking out the '0x'?

Reply:
@koderholic yes, apparently thorchain updated to remove 0x that's why we the api was failing

Reply:
Oh! Got it!